docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,720 @@
+      "                      '111110101': 0.034938562148434216,\n",
+      "                      '111110110': 0.09110862335695782,\n",
+      "                      '111111101': 0.019136638615493577,\n",
+      "                      '111111110': 0.03125},\n",
+      "    'eigenvalue': -1.3976638183589043,\n",
+      "    'optimal_parameters': {   ParameterVectorElement(θ[0]): 2.4172136410249263,\n",
+      "                              ParameterVectorElement(θ[1]): 0.1300326023040021,\n",
+      "                              ParameterVectorElement(θ[2]): 0.15993019481009985,\n",
+      "                              ParameterVectorElement(θ[3]): 3.7390997537834707,\n",
+      "                              ParameterVectorElement(θ[4]): 0.8097465551468879,\n",
+      "                              ParameterVectorElement(θ[5]): 1.9615965211249644,\n",
+      "                              ParameterVectorElement(θ[6]): -0.630630143145122,\n",
+      "                              ParameterVectorElement(θ[7]): 0.23897472147947907,\n",
+      "                              ParameterVectorElement(θ[8]): -0.13886785224051298,\n",
+      "                              ParameterVectorElement(θ[9]): -3.0852751354288372,\n",
+      "                              ParameterVectorElement(θ[10]): -0.14285829964838806,\n",
+      "                              ParameterVectorElement(θ[11]): 3.2556505722321214,\n",
+      "                              ParameterVectorElement(θ[12]): 2.3333085773562106,\n",
+      "                              ParameterVectorElement(θ[13]): -2.5910962466232923,\n",
+      "                              ParameterVectorElement(θ[14]): 4.098235006235481,\n",
+      "                              ParameterVectorElement(θ[15]): 1.4780964168602555,\n",
+      "                              ParameterVectorElement(θ[16]): 2.602834243044125,\n",
+      "                              ParameterVectorElement(θ[17]): -2.4240297705950544},\n",
+      "    'optimal_point': array([ 2.41721364, -0.1428583 ,  3.25565057,  2.33330858, -2.59109625,\n",
+      "        4.09823501,  1.47809642,  2.60283424, -2.42402977,  0.1300326 ,\n",
+      "        0.15993019,  3.73909975,  0.80974656,  1.96159652, -0.63063014,\n",
+      "        0.23897472, -0.13886785, -3.08527514]),\n",
+      "    'optimal_value': -1.3976638183589043,\n",
+      "    'optimizer_evals': 50,\n",
+      "    'optimizer_time': 30.188204050064087}\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.library import RealAmplitudes\n",
+    "from qiskit.algorithms.optimizers import COBYLA\n",
+    "from qiskit.algorithms import NumPyMinimumEigensolver, VQE\n",
+    "from qiskit.opflow import PauliExpectation, CVaRExpectation\n",
+    "from qiskit import execute, Aer\n",
+    "\n",
+    "# set classical optimizer\n",
+    "optimizer = COBYLA(maxiter=50)\n",
+    "\n",
+    "# set variational ansatz\n",
+    "ansatz = RealAmplitudes(7, reps=1)\n",
```

Comment:
```suggestion
    "ansatz = RealAmplitudes(reps=1)\n",
```
We should not need to set number of qubits when building out anzatzes. That was the whole rationale behind blueprint ccts to set it later once the size was known from the operator. VQE does this when one calls to compute the min eig value - and in this case its overwritten to '9' which is the size of the operator. So setting 7 here is misleading and confusing I think. Better not to do it at all - if one has to do it set it to the qubit operator size so its consistent - I would leave it unset though as this is how we would normally expect it to be used.

Reply:
Good catch. This was a leftover from previous tests. Now is updated.